### PR TITLE
Support for show/hide neutral losses spectrum annotation

### PR DESCRIPTION
**Pull request by rita-gwen**
Replaced the context menu items for ion type and charge selection with specialized menu-hosted controls.

### CODE REVIEW

File: pwiz_tools/Skyline/SkylineGraphs.cs [hunk 1]
```diff
@@ -506,7 +506,7 @@ private void LoadLayoutLocked(Stream layoutStream)
             _listGraphDetections.ToList().ForEach(DestroyGraphDetections);
             DetectionsGraphController.GraphType = type;
 
-            FormUtil.OpenForms.OfType<FoldChangeForm>().ForEach(f => f.Close());
+            CollectionUtil.ForEach(FormUtil.OpenForms.OfType<FoldChangeForm>(), f => f.Close());
```
Comment (rita-gwen) [inline pwiz_tools/Skyline/SkylineGraphs.cs:509]: This is a strange change, I don't remember doing it. Might has come from a master merge.

**Comment by rita-gwen** [comment]
This PR is ready for the code review.

**Comment by rita-gwen** [comment]
This is how it looks with enough losses defined:

**Review by brendanx67 — CHANGES_REQUESTED** [review]
Found my review. Mostly pretty minor stuff, but needs a bit more testing.

**Comment by rita-gwen** [comment]
I was thinking how to address that extra border problem when a button on the control has focus. Do you think this style will work? It doesn't have borders at all.

**Pull request merged**